lib/wsfed.js

```diff
@@ -125,7 +125,7 @@ module.exports = function(options) {
   return function (req, res, next) {
     options.getPostURL(req.query.wtrealm, req.query.wreply, req, function (err, postUrl) {
       if (err) return next(err);
-      if (!postUrl) return res.send(400, 'postUrl is required');
+      if (!postUrl) return res.status(400).send('postUrl is required');
```

Comment:
FYI - I didn't change this because this might break customer using an old version of Express

Reply:
Yes, good point. I've reverted this change.